interpreter/cling/include/cling/Interpreter/CompilationOptions.h

```diff
@@ -86,7 +86,7 @@ namespace cling {
       CodeGeneration = 1;
       CodeGenerationForModule = 0;
       IgnorePromptDiags = 0;
-      OptLevel = 2;
+      OptLevel = 1;
```

Comment:
I think I've asked in previous PRs but how does O0, O1 and O2 affect ROOT's performance in general. A good approximation of this would be timing a bunch of tutorials. Let's say something from math and graphics.

Reply:
I am still hoping to see grafana report those :-) FYI:
```
$ time root.exe -l -b -q ./tutorials/dataframe/df102_NanoAODDimuonAnalysis.C
```
opt | user | sys
----|------|----
O0  | 115s | 18s
O1  | 82s  | 11s
O2  | 99s  | 15s
O3  | 80s  | 12s

I use sys as an estimate of the uncertainty, i.e. +/- 3s. Overall runtime is obviously a combination of JIT (incl optimizer) time and runtime of the jitted code, so there's no simple slope.

Reply:
I was mostly asking for things outside dataframe - as many of the operations in dataframe are hotspot.

Reply:
We do monitor quite a lot in grafana; happy to point you to the plots if you want!